app/ota_client.py

```diff
@@ -710,7 +910,8 @@ def error_callback(e):
                 downloader=self._download,
             )
 
-            with Pool() as pool:
+            _max_workder = min(3, os.cpu_count())
```

Comment:
I think this change is for the proxy server not to be congested, but If the number of sub ecu increased, is this value appropriate?

Reply:
I think it is OK, I have tried small scale test with 5 downloading program with more workers to request files from a single ota_proxy server, and it works as expected.